specs/linux/ipmi_lan.table

```diff
@@ -0,0 +1,19 @@
+table_name("ipmi_lan")
+description("IPMI LAN information.")
+schema([
+    Column("mc_name", TEXT, "Name of the MC"),
+    Column("mc_id", TEXT, "Device ID of the MC"),
```

Comment:
Ah yeah, that's a really good point. Definitely seems worth it to me to leave the labeling as you had originally intended!